packages/injectable/src/dependency-injection-container/createContainer.registration.test.js

```diff
@@ -16,22 +16,67 @@ describe('createContainer.registration', () => {
     expect(actual).toBe('some-injected-instance');
   });
 
-  it('throws an error with the key if the value behind has no default export', () => {
-    const context = new Map();
+  it('given injectable file with no default export, when auto-registering, throws with name of faulty file', () => {
+    const requireContextStub = Object.assign(
+      () => ({
+        notDefault: 'irrelevant',
+      }),
+      {
+        keys: () => ['./some.injectable.js'],
+      },
+    );
+
+    expect(() => createContainer(() => requireContextStub)).toThrowError(
+      /\.\/some\.injectable\.js/,
```

Comment:
Test is more documentative has more coverage by adding full expected error message.